Hivemind/Assets/Scripts/GameWorld.cs

```diff
@@ -45,43 +72,192 @@ public static ResourceNode FindNearestKnownResource(Vector3 antPosition, Resourc
         return closest;
     }
 
-    public static Storage GetStorage()
+    public Storage GetStorage()
     {
         return storage;
     }
 
-    public static void RemoveResource(ResourceNode resource)
+    public void SetStorage(Storage Storage)
     {
-        KnownResources.Remove(resource);
-        UnknownResources.Remove(resource);
+        storage = Storage;
     }
 
-    public static void SetStorage(Storage Storage)
+    public void AddNewResource(ResourceNode resource)
     {
-        storage = Storage;
+        ResourceList.Add(resource);
+    }
+
+    public void RemoveResource(ResourceNode resource)
+    {
+        ResourceList.Remove(resource);
+    }
+
+    public void AddNewAnt(Ant ant)
```

Comment:
Hoezo is dit add new ant? volgensmij is het beter als dit AddAnt() is. aangezien andere Add's ook niet "new" zijn

Reply:
renamed